sdk/transaction.go

```diff
@@ -9,22 +9,18 @@ import (
 	"context"
 	"fmt"
 	"net/http"
-
-	"github.com/proximax-storage/go-xpx-utils/net"
 )
 
 type TransactionService struct {
 	*service
 	BlockchainService *BlockchainService
 }
 
-// returns Transaction for passed transaction id or hash
-func (txs *TransactionService) GetTransaction(ctx context.Context, id string) (Transaction, error) {
+// returns Transaction for passed transaction group and hash
+func (txs *TransactionService) getTransaction(ctx context.Context, group string, id string) (Transaction, error) {
```

Comment:
Change `group` to designated group type instead of a string.

Reply:
I will add a common method for it, because we have a lot of transaction groups and regarding task we need to add just a few groups, not all )